documentation/sdk/docs/libraries/python/how_tos/simple_transaction/simple_transaction/_expected_output.mdx

```diff
@@ -0,0 +1,5 @@
7087cc4255f1ecb14dc78d06138f592', '0x225ac460d1f242b22878ee13538f9c9b254ca64947122fd23ac75c57ba5436cc', '0x6caf738a4f2662a6aaa1f5871ea8e945c0566b3e5360fd9f1c8d7b364d43b9ec', '0x7df576764b60e4f1c9c48dd71264b0188ea93224bbfcab8bf40f55e7d44c02ba', '0xc8ce0462dc172f7c76cd6d29817f54650bbaf3ce1038ac4a9d236b044921bcfb', '0x61700b71c762dab50d29f7abf0950727912279ec65d32507f07994cc678a6534', '0x77fcee05b89cd39e4b0b5f1b4574d129a95c8501ae15d486e26fc7ecdf38cb5d', '0x152544422fb02057b58ce9595f63bbffc5ad7131c8c3b4043fbb3fecd9c59fce', '0xa15dec712febbb756c3425f4af8affdd977d37eaad280d0e9dc21d769c778de5', '0x33a3eecd21508ede0a536b36f4d78ec3c5c45c51897eca0f9507e5c53f58f1fb', '0x2afe8d6253cc5cf991c2489f85ea9b322f6337e6f102e1341959a2bf96723184', '0xc0580e816bfb4160d28361f845e81154ac0a4eb6bd01ebe26171f06af7ac9312', '0x58b2759ccf4e916eaf1e2a216807b2647f492593190bcddd6425aa337658d592', '0xa900b828ea7b3c8ea79dd3486336cb02554b95213ac608087d3863400ec6fc0a', '0x15eee45273ed97bd3e0fabfbee7e27f1ce9739a04b008e82798f48881cfc85fd', '0x410b89aa6231ef61f1fc1a25d7f8142aa874e116ecd90aef5b515941f8cbd5fc', '0xd7a279100a21ceebd5705154f38e7846375b32256bbe7404c39fdbb111faf1f9', '0x2178e5c287e3cb0aa0d5fe519b557762717129ce3bc789142494d4f243aa2521', '0x21c2050f794b380a4aebea9e12bd6572e94cc57e028744c460c338840d605021', '0x779685f95a84d016ce8617817725fb0a2d8795e6d88763805e5686dd9df4a272', '0xc78407935e15c90263b56d50df6d376b1ae1ef0619807eda22a89895db3361b6', '0x5aca32921059a9851d3d19753966f4cc9cb725117df4765900843fd00000532d', '0x69d4239b3bca714fbc073d5261d9047aace9e2999718872c054d8ac8e4c25b80', '0x9e110b5bb9ad527cb3a058472c6e7eb7cb73fcbda6f47783b9f3f8d7c33325c2', '0xb95a8b112d97bc68ebd27dabb147a889632eac111be2ec997afad0f79f2c31c9', '0x3a9214572cec5ac1980b35559dbd333b79f045891ab8304c0de42f368af31565', '0xaaefd663969bda4c8409cdefcf5f50959c3ef4cf9e65b7ec45757c2cd1e2f940', '0x660f47feeff2e6043845ff114a95a00143c380df8c2ad073c9ae24e7ec4abf1f', '0xb9c95f815d409e68eee1c1e8d2c829d869951b62822a00a423af95f6bf09957a', '0x8787d20a248c33b3e3148f61725da400d6547a633b107af0720ca151d1df8390', '0x18e2474fbdea324ac7eae2c40d8b46f4533c9102bfd1f427aca5b1c78efc6813'], 'aliases': ['0x224e1b4e08f62f27d283707ecd2a2fa4d69b05dfdefd556b8504d33b1ee59965', '0x18e6594c75503153d795a1563050af5a84e278882b4aa5f17cca31dafab0efb1', '0x60a92b816d3b3206bd41e4b183a353e04c8c79dde3d2bb4bf2a4c98e5938d7e6', '0x5849d0dfeb2c0bf940660373d71181ac2bf92cdf8adb2b3085a1636720e6fbf9', '0x96717e6d19c13b1c5b120d60b23217f541b5b779e51212e01d72e7fa1f7090cf', '0xa5526c4a15558b709340822edf00cb348d8606a27e2e59b00432a0afe8afb74d', '0xf9c702ffe50c35d331b2df02295c2cc6d92f883530ff231bd76d1f6a72cb1d95', '0x97eb7a447cd62e1c373ff8188ba422f5c1b0687707d38e10e8366a1c20d33fea'], 'foundries': ['0x08a5526c4a15558b709340822edf00cb348d8606a27e2e59b00432a0afe8afb74d0200000000', '0x08a5526c4a15558b709340822edf00cb348d8606a27e2e59b00432a0afe8afb74d0100000000', '0x0818e6594c75503153d795a1563050af5a84e278882b4aa5f17cca31dafab0efb10100000000'], 'potentiallyLockedOutputs': {}}
```

Comment:
Maybe the expected output should be just some amount (by using a new mnemonic) and not all the native tokens and NFTs :trollface:

Reply:
I just copied your existing examples :trollface: 
Hence the question in my description above: 
> Should I try to also unify the examples if it's not too much work, as I prefer having examples over not having any ;)